src/test/java/com/wikia/webdriver/common/core/imageutilities/ImageComparison.java

```diff
@@ -57,7 +59,8 @@ public boolean isColorImage(BufferedImage image, Color color, int accuracy) {
     int diffCount = 0;
     for (int x = 0; x < image.getWidth(); x++) {
       for (int y = 0; y < image.getHeight(); y++) {
-        if (image.getRGB(x, y) != color.getRGB()) {
+        Color pixelColor = new Color(image.getRGB(x,y));
```

Comment:
LGTM in general.
There might be performance/less memory consumption gain by declaring Color pixelColor outside of the loops and then just updating it, however I'm not sure about it - maybe it is optimization that JVM is able to do itself. Not a merge blocker in my opinion.